src/storage/src/hummock/utils.rs

```diff
@@ -83,15 +84,15 @@ pub fn validate_table_key_range(version: &HummockVersion) {
     }
 }
 
-pub fn filter_single_sst<R, B>(info: &SstableInfo, table_id: TableId, table_key_range: &R) -> bool
+pub fn filter_single_sst<R, B>(info: &SstableInfo, table_id: &TableId, table_key_range: &R) -> bool
```

Comment:
`TableId` implements `Copy` (it is only a wrapper for `u32`), so I think it's better to pass an owned value in case that it causes some lifetime issue.

Reply:
fixed.